### PR TITLE
Release v1.67.0

### PR DESCRIPTION
**Pull request by shivaspeaks**

### CODE REVIEW

**Comment by ejona86** [comment]
The "base" for this PR should be v1.67.x, not the random branch "release-v1.67.0". I changed the base and that seems to have resolved the file conflicts.

File: README.md [hunk 1]
```diff
@@ -79,18 +79,18 @@ Download [the JARs][]. Or for Maven with non-Android, add to your `pom.xml`:
...
 `grpc-protobuf-lite` instead of `grpc-protobuf`:
 ```gradle
-implementation 'io.grpc:grpc-okhttp:1.66.0'
-implementation 'io.grpc:grpc-protobuf-lite:1.66.0'
-implementation 'io.grpc:grpc-stub:1.66.0'
+implementation 'io.grpc:grpc-okhttp:1.67.0'
+implementation 'io.grpc:grpc-protobuf-lite:1.67.0'
+implementation 'io.grpc:grpc-stub:1.67.0'
 compileOnly 'org.apache.tomcat:annotations-api:6.0.53' // necessary for Java 9+
 ```
```
Comment (ejona86) [inline README.md:96]: "the JARs" link below also has a reference to the old 1.66.0. You may want to apply this as-is and just have that minor issue in the release, but then fix it on master and the v1.67.x branch after-the-fact.

**Review by ejona86 — APPROVED** [review]
Reminder: DO NOT MERGE FROM GITHUB. Follow the documented release steps.

File: README.md (hunk 1 above)
Comment (shivaspeaks) [reply README.md:96]: Or should I do a new commit making it 1.67.x?

Comment (ejona86) [reply README.md:96]: If you just add another commit, it won't be in the release anyway. You could delete your local v1.66.0 tag and fix the "Update README etc to reference 1.67.0" commit, but I sort of expect that to lead to far more problems for your first release.

Comment (shivaspeaks) [reply README.md:96]: Alright! Sounds good.

**Pull request merged**